plugin/src/main/java/net/dzikoysk/funnyguilds/config/MessageConfiguration.java

```diff
@@ -362,14 +362,16 @@ public class MessageConfiguration extends OkaeriConfig {
     public String inviteAllyAmount = "&7Osiagnieto juz &cmaksymalna &7liczbe sojuszy miedzygildyjnych! (&c{AMOUNT}&7)";
     @Comment("Dostępne zmienne: {AMOUNT}, {GUILD}, {TAG}")
     public String inviteAllyTargetAmount = "&7Gildia {TAG} posiada juz maksymalna liczbe sojuszy! (&c{AMOUNT}&7)";
-    public String inviteCancelled = "&cCofnieto zaproszenie!";
+    public String inviteCancelled = "&cCofnieto zaproszenie dla {USER}!";
```

Comment:
Dodaj komentarz, że można użyć placeholderu {USER}

Reply:
Nie ma co tutaj dodawać, bo to do oddzielnego PR powinno iść. :trollface: